### ISSUE TITLE
Feed for sklearn's stochastic gradient descent

### ISSUE THREAD

**Issue by maarten911** [boing102/CI-Project#8]
I don't succeed to set a seed. Without, it's difficult to compare different to compare different approaches (pca, normalizing data)
